src/iperf_error.c

```diff
@@ -548,6 +548,9 @@ iperf_strerror(int int_errno)
             snprintf(errstr, len, "unable to set/get socket keepalive TCP number of retries (TCP_KEEPCNT) option");
             perr = 1;
             break;
+        case IEMAXSERVERDURATIONEXCEEDED:
+            snprintf(errstr, len, "client's max duration exceeds the server's maximum permitted duration");
```

Comment:
Suggest to replace "duration" with "test duration".

Reply:
Changed